cli/src/merge_tools/external.rs

```diff
@@ -50,6 +53,15 @@ pub struct ExternalMergeTool {
     pub merge_tool_edits_conflict_markers: bool,
 }
 
+#[derive(clap::ValueEnum, serde::Deserialize, Copy, Clone, Debug, Eq, PartialEq)]
```

Comment:
nit: `clap::ValueEnum` is no longer needed

Reply:
done